packages/core/lib/commands/call/meta.js

```diff
@@ -0,0 +1,82 @@
+module.exports = {
+  command: "call",
+  description: "Call read-only contract function with arguments",
+  builder: {
+    "url": {
+      describe: "Use specified URL for ethereum provider",
+      type: "string"
+    },
+    "fetch-external": {
+      describe: "Fetch referenced verified contracts as needed",
+      alias: "x",
+      type: "boolean",
+      default: false
+    },
+    "block-number": {
+      describe: "Calls specified contract function at specific block",
+      alias: "b",
+      type: "integer",
+      default: "latest"
+    },
+    "contract-address": {
+      describe: "The contract address to be called",
+      type: "string"
+    },
+    "contract-name": {
+      describe: "The contract name to be called",
+      type: "string"
+    }
+  },
+  help: {
+    usage:
+      "truffle call <contract-address>|<contract-name> <function-name>|<function-signature>\n" +
+      "                             " + // spacing to align with previous line
+      "[<arg>...] [--fetch-external|-x] [--network <network>|--url <provider_url>]\n" +
+      "                             " + // spacing to align with previous line
+      "[--block-number|-b <block_number>]",
+    options: [
+      {
+        option: "<contract-name>",
+        description: "The contract name to be called."
+      },
+      {
+        option: "<contract-address>",
+        description: "The contract address to be called.\n"
+      },
+      {
+        option: "<function-name>",
+        description:
+          "The function name inside the specified contract to be called."
+      },
+      {
+        option: "<function-signature>",
+        description:
+          "The function signature inside the specified contract to be called."
```

Comment:
Does this mean that people can think of "function signature" as "function selector" which is the hash of the function signature? In that case, explicitly specifying it in the description to avoid confusion makes sense.

Reply:
Yes, "signature" is unfortunately sometimes abused to mean "selector".